dev/jetbrains-test/BUILD.yaml

```diff
@@ -17,3 +17,19 @@ scripts:
     script: |
         cp $COMPONENTS_IDE_JETBRAINS_GATEWAY_PLUGIN__PUBLISH_LATEST/build/distributions/gitpod-gateway.zip gitpod-gateway.zip
         ./test.sh
+  - name: install-gui-dependencies
+    script: |
+      sudo install-packages \
```

Comment:
should it be part of our dev dockerimage instead?

Reply:
It's not necessary, I just need this script to debug code instead of wait GHA to do it